pkg/render/dex_config.go

```diff
@@ -119,27 +121,28 @@ type DexRelyingPartyConfig interface {
 
 func NewDexRelyingPartyConfig(
 	authentication *oprv1.Authentication,
-	tlsSecret *corev1.Secret,
+	caSecret *corev1.Secret,
 	dexSecret *corev1.Secret,
 	clusterDomain string) DexRelyingPartyConfig {
-	return &dexRelyingPartyConfig{baseCfg(authentication, tlsSecret, dexSecret, nil, clusterDomain)}
+	return &dexRelyingPartyConfig{baseCfg(nil, authentication, nil, dexSecret, nil, caSecret, clusterDomain)}
 }
 
 func NewDexKeyValidatorConfig(
 	authentication *oprv1.Authentication,
-	tlsSecret *corev1.Secret,
+	caSecret *corev1.Secret,
 	clusterDomain string) DexKeyValidatorConfig {
-	return &dexKeyValidatorConfig{baseCfg(authentication, tlsSecret, nil, nil, clusterDomain)}
+	return &dexKeyValidatorConfig{baseCfg(nil, authentication, nil, nil, nil, caSecret, clusterDomain)}
 }
 
 // Create a new DexConfig.
 func NewDexConfig(
+	installation *oprv1.InstallationSpec,
```

Comment:
Did you consider passing in the CertificateManagement instead of the full installation? I'm not sure if that would be better.